includes/wikia/models/NavigationModel.class.php

```diff
@@ -556,16 +522,16 @@ public function parseOneLine($line) {
 			} else {
 				if ( empty( $link ) ) {
 					$href = '#';
-				} else if( $link{0} == '#' ) {
+				} else if ( $link{0} == '#' ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This branch's code block is the same as the block for the branch on line 523. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1871)